browser/ui/tabs/split_view_browser_data.cc

```diff
@@ -29,6 +30,12 @@ void SplitViewBrowserData::TileTabs(const Tile& tile) {
   CHECK(!IsTabTiled(tile.first));
   CHECK(!IsTabTiled(tile.second));
 
+  if (!is_testing_) {
```

Comment:
Is it okay to violate this in tests?

Reply:
Yeah, in the_split_view_browser_data.unittest, browser doesn't exist at all.